### DISCUSSION TITLE
Authentication to Local AD

### DISCUSSION THREAD

**Discussion by hharris75** [AdhocAdam/smletsexchangeconnector#482] [Q&A]
It seems this only works with cloud only accounts and not Local AD accounts synced to the cloud that authenticate thru ADFS. Is this as designed?
Thanks

**Comment by AdhocAdam** [comment]
The connector authenticates to Exchange utilizing Exchange Web Services for on premise and hosted 365 deployments.

There is nothing in the connector that explicitly handles or interacts with ADFS. If there is a specific error message your seeing I'm certainly open to looking at to see if I can provide further insight.

**Reply by hharris75** [reply]
After some testing it seems the issue is with Azure App Registration and ADFS. For it to work the workflow account needs to be authenticated in the cloud. I tried just getting a token with PS and get a bad password or username error. 

So, you either have to create a cloud only account or I added the workflow account to our Staged Rollout for Password Hash Sync.
